js/filter.js

```diff
@@ -9,7 +9,7 @@ const OP_ALIAS = {
     "<=": "lte"
 };
 
-const KEYWORDS = {
+export const KEYWORDS = {

```

Comment:
Given this is now exportable I'd rename to something that makes sense from the "outside" (e.g. `FILTER_KEYWORDS`).